app/components/flash_message_component.html.erb

```diff
@@ -1,32 +1,4 @@
 <% if !flash.empty? %>
-  <% if flash[:warning] %>
-    <div class="app-banner app-banner--warning" aria-labelledby="warning-message">
```

Comment:
We've got a few banners which could potentially be moved into the new structure, example: 
https://github.com/DFE-Digital/apply-for-teacher-training/blob/0e14c10c27bb5a694e6eec1af92f9e26a9a14834/app/components/candidate_interface/ucas_downtime_component.html.erb
Are these good candidates/possible to move into the new `NotificationMessageComponent` components or are they different?

Reply:
I missed this one. I wonder why this is using a warning rather than an info box 🤔

Reply:
Addressed. Also updated another three entries related to displaying information to the user when the DFE Sign in has problems/is down and removed the `warning`, `info` and `success` states from banner, since we no longer use them.